### ISSUE TITLE
Retire Custom Dashboard Provisioning

### ISSUE THREAD

**Issue by tomjn** [Varying-Vagrant-Vagrants/VVV#1946]
Right now, there's a special provisioner for the dashboard, with config values to customize it. I believe this is unnecessary, and I'd like to replace it with the following:

 - always provision the official dashboard
 - a custom dashboard can just be another site and use the same mechanism, except without the WP install bit, an adjusted nginx config, custom hostname, and some PHP in the `public_html` folder. No reason it couldn't be something like `dashboard.test`
 - a config option to change the dashboard URL so the user gets redirected to `dashboard.test` or wherever on demand

This greatly simplifies the whole process of creating a dashboard, reduces the needed documentation greatly, and makes it possible to have multiple dashboards at once.

**Comment by msaggiorato** [comment]
Would you do a phase out of the dashboard? Still allowing a custom dashboard to be provisioned with the dashboard setting, but displaying a warning to the user that doing this is deprecated?

**Comment by tomjn** [comment]
No we'd just provison the official dashboard and give it a proper site provisioner script like the site templates. I'd even argue to move it out of a synced folder. People can set up custom dashboards the same way they set up other sites

**Comment by Mte90** [comment]
So maybe the idea is to convert https://github.com/Varying-Vagrant-Vagrants/VVV/blob/develop/provision/provision-dashboard.sh as vvv-utilities?

**Comment by tomjn** [comment]
Not really, we'd still provision it the same way, we'd just remove all the options to configure it.

Then, if people want a custom dashboard, there's nothing stopping them creating a `dashboard.test` site and rolling their own or running a fork
